src/main/scala/com/yotpo/metorikku/configuration/job/output/File.scala

```diff
@@ -4,4 +4,5 @@ package com.yotpo.metorikku.configuration.job.output
 case class File(dir: String,
                 checkpointLocation: Option[String]) {
   require(Option(dir).isDefined, "Output file directory: dir is mandatory.")
+  val absolutePath: String = new java.io.File(dir).getAbsoluteFile.toString
```

Comment:
This can be S3... you can't really do that...

Reply:
your right.... i dont know how to solve it then....